readme.md

```diff
@@ -130,4 +86,42 @@ Specifying `--js` tells the generator to use JavaScript.
 
 >**Note:** Do not use this flag when you pass `react` as framework argument.
 
+## Running the Generated Site
+
+Office Add-ins must be hosted in an HTTPS site. Yo Office generates a [Browsersync](https://browsersync.io/) configuration file (`bsconfig.json`) that references a self-signed certificate. Your computer will need to trust the certificate before you can use the generated add-in. 
+
+> **Important:** Before you continue with this Readme, take the following steps (Windows):
+>
+>    1.	Go to {project root}\certs.
+>    2.	Double-click ca.crt, and select **Install Certificate**.
+>    3.	Select **Local Machine** and select **Next** to continue.
+>    4.	Select **Place all certificates in the following store** and then select **Browse**.
+>    5.	Select **Trusted Root Certification Authorities** and then select **OK**.
+>    6.	Select **Next** and then **Finish**.
+
+  		  
+Launch the local HTTPS site on `https://localhost:3000` by simply typing the following command in your console:
+
+```bash
+$ npm start
+```
+
+Browsersync will start an HTTPS server, which includes a self-signed SSL cert that your development environment must trust. Refer to our doc, [Adding Self-Signed Certificates as Trusted Root Certificate](src/docs/ssl.md), for instructions on how to do this.
+
+Browse to the 'External' IP address listed in your console to test your web app across multiple browsers and devices that are connected on your local network.

```

Comment:
Seems like the "Adding Self-Signed Certificates as Trusted Root Certificate" page contains guidance for the same task that you've enumerated above (in steps 1-6) -- except that the guidance/instructions in that article is different than steps 1-6 above.   Perhaps consider updating the "Adding Self-Signed Certificates as Trusted Root Certificate" page with correct/complete guidance for all platforms, continue linking to that page from this README, and remove steps 1-6 above (i.e., the **Important** alert) from this article?

Reply:
I suspect you can do this a couple of different ways. I'm inclined to leave the the steps in the readme and point to the "Adding Self-signed..." page as the instructions for Mac.